README.md

```diff
@@ -5,35 +5,37 @@ genymotion-binocle
 ![Binocle](../master/binocle/src/main/res/drawable-xxxhdpi/ic_launcher.png?raw=true)
 
 
-This is a simple Android app with Android tests demonstrating how to use Genymotion Java API to check the behavior of your app when sensors values change.
+Binocle is a simple application showcasing Genymotion Java API use.
+In this application, you can find activities for which the behavior depends on sensor values.
+Below are some Android test examples built with Genymotion Java API to manipulate sensor values and check activity behaviors.
 
 [Genymotion Java Api documentation](https://cloud.genymotion.com/page/api/)    
 [Genymotion Java Api Javadoc](https://cloud.genymotion.com/static/external/javadoc/index.html)
 
 ### Battery
-This activity displays a warning if the device is not plugged to a power source and has got less than 10% power left.  
-Here is the Fragment showing it: [BatterySampleFragment.java](https://github.com/Genymobile/genymotion-binocle/blob/master/binocle/src/main/java/com/genymotion/binocle/BatterySampleFragment.java)  
-Here is an androidTest testing the behavior: [TestBattery.java](https://github.com/Genymobile/genymotion-binocle/blob/master/binocle/src/instrumentTest/java/com/genymotion/binocle/test/TestBattery.java)
+An application must display a warning if the device is not plugged to a power source and has less than 10% of charge left.  
```

Comment:
This part of the application displays a warning ...

An application must display me parait un truc générique à n'importe quelle appli et ce n'est pas le cas ici. C'est bien un dev spécifique qu'on a fait